src/main/java/net/notfab/lindsey/core/commands/wiki/Apoiase.java

```diff
@@ -0,0 +1,114 @@
+            return false;
+        }
+        String name = "";
+        OkHttpClient client = new OkHttpClient();
+        EmbedBuilder embed = new EmbedBuilder();
+        Request request = new Request.Builder()
+            .url("https://apoia.se/api/v1/users/campaign?page=0&q=" + this.argsToString(args, 0))
+            .get()
+            .build();
+        Response resp = client.newCall(request).execute();
+        JSONObject obj = new JSONObject(resp.body().string());
+        try {
+            name = obj.getJSONArray("campaigns").getJSONObject(0).getJSONObject("campaign").getString("slug");
+        } catch (Exception e) {
+            msg.send(channel, i18n.get(member, "core.not_found", this.argsToString(args, 0)));
+            return false;
+        }
+        request = new Request.Builder()
+            .url("https://apoia.se/api/v1/users/" + name)
+            .get()
+            .build();
+        resp = client.newCall(request).execute();
+        obj = new JSONObject(resp.body().string());
+        JSONObject campaigns = obj.getJSONArray("campaigns").getJSONObject(0);
+        JSONObject state = obj.getJSONArray("address").getJSONObject(0);
+        boolean nsfw = false;
+        if (campaigns.getBoolean("explicit")) {
+            nsfw = true;
+        }
+        if (nsfw && !channel.isNSFW()) {
+            msg.send(channel, i18n.get(member, "core.not_nsfw"));
+            return false;
+        }
+        embed.setTitle(campaigns.getString("name"), "https://www.apoia.se/" + campaigns.getString("slug"));
+        if (!campaigns.getJSONObject("about").isNull("slogan")) {
+            embed.setDescription(campaigns.getJSONObject("about").getString("slogan"));
+        }
+        embed.setFooter(i18n.get(member, "commands.fun.anime.request") + " " + member.getEffectiveName() + "#" + member.getUser().getDiscriminator(),
+            member.getUser().getEffectiveAvatarUrl());
+        embed.setImage(campaigns.getJSONObject("about").getString("photo"));
+        embed.addField(i18n.get(member, "commands.wiki.crowdfunding.category"), campaigns.getJSONArray("categories").getString(0), true);
+        if (!campaigns.getJSONArray("goals").isEmpty()) {
+            embed.addField(i18n.get(member, "commands.wiki.crowdfunding.goal"), "R$" + campaigns.getJSONArray("goals").getJSONObject(0).getInt("value"), true);
+        }
+        embed.addField(i18n.get(member, "commands.wiki.crowdfunding.creator"), obj.getString("username"), true);
+        embed.addField(i18n.get(member, "commands.wiki.crowdfunding.createdDate"), campaigns.getString("createdDate").split("T")[0], true);
+        embed.addField(i18n.get(member, "commands.wiki.crowdfunding.city"), state.getString("city") + " - " + state.getString("state"), true);
+        embed.addField(i18n.get(member, "core.nsfw"), Boolean.toString(nsfw), true);
+        if (campaigns.getString("status").equals("published")) {
+            embed.setColor(java.awt.Color.GREEN);
```

Comment:
Please use either GFXUtils (from HEX or one of the default values) or at least use the import directly.